### PR TITLE
Add solution

### PR DESCRIPTION
**Pull request by ssaviak**
[DEMO LINK](https://ssaviak.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,32 @@
...
+
+      {
+        age
+          ? (
+            <p className="Person__age">
+              {age && `I am ${age}`}
+            </p>
+          )
+          : null
+      }
```
Comment (olya-shyian) [inline src/components/Person/Person.jsx:21]: you can write simplier like 
```
{age && (
  <p className="Person__age">
    {`I am ${age}`}
  </p>
)}
      ```

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,32 @@
...
+
+      <p className="Person__partner">
+        {
+          isMarried
```
Comment (olya-shyian) [inline src/components/Person/Person.jsx:25]: you can leave `isMarried` in the same line with `{` bracket like

```
 {isMarried
    ?
```

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import './Person.css';
+
+export function Person({ person }) {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const isMan = sex === 'm';
```
Comment (olya-shyian) [inline src/components/Person/Person.jsx:5]: you can improve this check , istead of `isMan` just find which one should be
``` const partner = sex === 'm' ? 'wife' : 'husband';```

File: src/components/Person/Person.jsx [hunk 4]
```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import './Person.css';
+
+export function Person({ person }) {
+  const { name, age, sex, isMarried, partnerName } = person;
```
Comment (olya-shyian) [inline src/components/Person/Person.jsx:4]: I'm glad that you use destructuring here) well done)

**Review by olya-shyian — CHANGES_REQUESTED** [review]
Superb! Nice work!🚀

**Review by YegorKorenkov — APPROVED** [review]
Gooood job!

**Pull request closed without merging**